AppCenterDistribute/AppCenterDistribute/MSDistribute.m

```diff
@@ -269,8 +269,8 @@ + (MSUpdateTrack)updateTrack {
   return [MSDistribute sharedInstance].updateTrack;
 }
 
-+ (void)configure:(MSDistributeFlags)flags {
-  [[MSDistribute sharedInstance] configure:flags];
++ (void)disableAutomaticCheckForUpdates {
+  [sharedInstance disableAutomaticCheckForUpdates];
```

Comment:
'sharedInstance' is not initialized if this method is called as the first call to MSDistribute.
```suggestion
  [[MSDistribute sharedInstance] disableAutomaticCheckForUpdates];
```